packages/stacked_generator/lib/src/generators/getit/stacked_locator_content_generator.dart

```diff
@@ -15,7 +15,7 @@ class StackedLocatorContentGenerator extends BaseGenerator {
   });
   @override
   String generate() {
-    writeLine("// ignore_for_file: public_member_api_docs");
+    writeLine("// ignore_for_file: public_member_api_docs, unused_import");
```

Comment:
Why did you add ", unused_import"

Reply:
I added it before the imports was fixed I will remove it